consensus/merge/src/main/java/org/hyperledger/besu/consensus/merge/blockcreation/MergeBlockCreator.java

```diff
@@ -60,7 +59,7 @@ public class MergeBlockCreator extends AbstractBlockCreator {
         parentHeader);
   }
 
-  public Block createBlock(
+  public BlockCreationResult createBlock(
```

Comment:
Not at the moment. However, I fill this may open window to reuse block processing results. Maybe this is somehow related to this performance improvement [here](https://github.com/hyperledger/besu/compare/main...ahamlat:besu:cache-validation-block?expand=1)